it/controllers/QuestionControllerISpec.scala

```diff
@@ -47,21 +45,6 @@ class QuestionControllerISpec extends BaseISpec with LogCapturing with BaseOneSe
       questions.map(q => q.questionKey) should contain(paymentToDateQuestion.questionKey)
     }
 
-//    "return 200 and a sequence of non p60 question if provided with valid json but P60 returns not found" in new Setup {
```

Comment:
Just curious - why is this test commented out?

Reply:
I believe it was copied and pasted from IV